src/Compilers/Core/Portable/InternalUtilities/OneOrMany.cs

```diff
@@ -17,7 +18,6 @@ namespace Roslyn.Utilities
     /// Used when a collection usually contains a single item but sometimes might contain multiple.
     /// </remarks>
     internal readonly struct OneOrMany<T>
-        where T : notnull
```

Comment:
overly restrictive and not necessary.  this also blocked OneOrMany from being usable in NodeStateTable.